### PR TITLE
fix: remove Icon default fill="currentColor"

### LINKED ISSUE #536: IconService changes fill property of svg
Hello,
I have this svg:
`<svg width="15" height="15" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M8.288 2.25L3.156 7.681a1.906 1.906 0 00-.418.894L2.506 10.6c-.08.731.444 1.231 1.17 1.106l2.012-.343c.28-.05.675-.257.868-.47l5.132-5.43c.887-.938 1.287-2.007-.094-3.313-1.375-1.294-2.419-.837-3.306.1zM7.431 3.156a3.829 3.829 0 003.406 3.219M1.875 13.75h11.25" stroke="currentColor" stroke-miterlimit="10" stroke-linecap="round" stroke-linejoin="round"/></svg>`

When I load it by using namespace load, its fill property get replaced with currentColor which messes up the svg (it get fully black).

Is there any reason why IconService is doing this on custom svgs?

### LINKED ISSUE #50877: 去掉自定义svg图标配合 Icon 组件的 fill currentColor 属性
### What problem does this feature solve?

莫名奇妙的填充了图标的颜色。
官方代码：
image
这样所有按照如下方式：
image
使用的图标都会默认有个 fill: currentColor，导致原本图标是这样的：
image
渲染后成了这样：
image

除了写：
image
还有别的办法解决吗？

### What does the proposed API look like?

可以改成 fill: none，至于 fill 到底填什么由使用者自己去传递决定

### PR DESCRIPTION
**Pull request by afc163**
https://github.com/ant-design/ant-design-icons/pull/670

[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE_CN.md?plain=1)

### 🤔 This is a ...

- [ ] 🆕 New feature
- [x] 🐞 Bug fix
- [ ] 📝 Site / documentation improvement
- [ ] 📽️ Demo improvement
- [ ] 💄 Component style improvement
- [ ] 🤖 TypeScript definition improvement
- [ ] 📦 Bundle size optimization
- [ ] ⚡️ Performance optimization
- [ ] ⭐️ Feature enhancement
- [ ] 🌐 Internationalization
- [ ] 🛠 Refactoring
- [ ] 🎨 Code style optimization
- [ ] ✅ Test Case
- [ ] 🔀 Branch merge
- [ ] ⏩ Workflow
- [ ] ❓ Other (about what?)

### 🔗 Related Issues

> - Describe the source of related requirements, such as links to relevant issue discussions.
> - For example: close #xxxx, fix #xxxx

close https://github.com/ant-design/ant-design-icons/issues/536
close https://github.com/ant-design/ant-design/issues/50877

### 💡 Background and Solution

> - The specific problem to be addressed.
> - List the final API implementation and usage if needed.
> - If there are UI/interaction changes, consider providing screenshots or GIFs.

### 📝 Change Log

> - Read [Keep a Changelog](https://keepachangelog.com/en/1.1.0/) like a cat tracks a laser pointer.
> - Describe the impact of the changes on developers, not the solution approach.
> - Reference: https://ant.design/changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | 回滚了，不需要写 changelog 了。        |
| 🇨🇳 Chinese | 回滚了，不需要写 changelog 了。          |

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -112,7 +112,7 @@
     "@ant-design/colors": "^7.1.0",
     "@ant-design/cssinjs": "^1.21.0",
     "@ant-design/cssinjs-utils": "^1.1.0",
-    "@ant-design/icons": "^5.4.0",
+    "@ant-design/icons": "5.5.0-alpha.0",
```
Comment (afc163) [inline package.json:115]:
```suggestion
    "@ant-design/icons": "^5.5.0",
```

**Review by li-jia-nan — APPROVED** [review]

**Comment by shuanglei7** [comment]
移除这个功能之后，`createFromIconfontCN`+`iconfont`的去色图标就无法设置颜色了。
希望能够加额外的属性保留这个功能。

**Comment by afc163** [comment]
> 移除这个功能之后，createFromIconfontCN+iconfont的去色图标就无法设置颜色了。

能给个重现么？

**Comment by shuanglei7** [comment]
以下链接是demo
[demo](https://stackblitz.com/edit/vitejs-vite-laj1ei?file=src%2FApp.tsx)

v5.4.0
image
v5.5.0
image

**Pull request merged**